src/line-chart/line-chart.component.ts

```diff
@@ -175,7 +178,6 @@ export class LineChartComponent extends BaseChartComponent {
   @Input() yAxisTickFormatting: any;
   @Input() roundDomains: boolean = false;
   @Input() tooltipDisabled: boolean = false;
-  @Input() showSeriesOnHover: boolean = true;
```

Comment:
Because the circles are not there any more. There is only one circle per series now, and it gets positioned based on where the mouse is. Before 6.0.0, we had a circle component created for every data point.